.github/workflows/execute-all-notebooks.yml

```
@@ -0,0 +1,92 @@
+
+  execute-all-notebooks:
+    needs:
+      - launch-ec2-runner
+    runs-on: ${{ needs.launch-ec2-runner.outputs.label }}
+
+    permissions:
+      pull-requests: write
+
+    steps:
+      - name: Setup Environment
+        run: |
```

actionlint (1.7.8)

property "instance-id" is not defined in object type {ec2-instance-id: string; label: string}

(expression)